components/onboarding_tasks/constants.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const FINISHED = 999;
+
+export const OnboardingTaskCategory = 'on_boarding_task_list';
+
+// Whole task list is based on these
+export const OnboardingTasksName = {
+    CHANNELS_TOUR: 'channels_tour',
+    BOARDS_TOUR: 'boards_tour',
+    PLAYBOOKS_TOUR: 'playbooks_tour',
+    INVITE_PEOPLE: 'invite_people',
+    DOWNLOAD_APP: 'download_app',
+    COMPLETE_YOUR_PROFILE: 'complete_your_profile',
+    VISIT_SYSTEM_CONSOLE: 'visit_system_console',
+};
+
+export const OnboardingTaskList = {
+    ONBOARDING_TASK_LIST_OPEN: 'on_boarding_task_list_open',
+    ONBOARDING_TASK_LIST_SHOW: 'on_boarding_task_list_show',
+    ONBOARDING_VIDEO_MODAL: 'on_boarding_video_modal',
+};
```

Comment:
Nice changes! That is a lot of renames. Could we also change these last string values to `onboarding`? Looks like there's one left in `utils/constants.jsx`, too.

Reply:
Yeah sure